data/132/706/155/9/1327061559.geojson

```diff
@@ -3,7 +3,9 @@
   "type": "Feature",
   "properties": {
     "edtf:cessation":"uuuu",
+    "edtf:deprecated":"2024-01-19",
```

Comment:
Nit here and elsewhere, `"2024-01-25"` as time marches on (and otherwise it's a noticeable gap with the `wof:lastmodified` time stamp).